src/resource-descriptions/people.js

```diff
@@ -2,5 +2,40 @@ module.exports = {
   urlTemplates: {
     "self": "http://127.0.0.1:3000/people/{id}",
     "relationship": "http://127.0.0.1:3000/people/{ownerId}/relationships/{path}"
+  },
+
+  /**
+   * An optional function called on each resource provided by the client (i.e.
+   * in a POST or PATCH request) before it's sent to the adapter for saving. You
+   * can transform the data here as necessary or pre-emptively reject the
+   * request.
+   *
+   * @param {Resource} resource - The resource to be transformed
+   * @param {Object} req - The framework's request object
+   * @param {Object} res - The framework's response object
+   * #param {Funcation} superFn - The Super function
```

Comment:
Can you make this line something like: 

```
@param {function|undefined} superFn - The beforeSave function on the parent resource type, if any
```

That fixes a couple typos (i.e. @ instead of #, and the spelling of function), and also notes that superFn can be undefined.